book/src/platformer/03_importing_tiled_levels.md

```diff
@@ -0,0 +1,325 @@
+# Importing Tiled levels
+
+What we want to write is something that parses the Tiled level and creates a representation in Rust.
+This will be run at build time.
+We do this at build time to avoid doing any parsing on the GBA itself.
+
+There is a Rust crate for parsing Tiled levels called [`tiled`](https://crates.io/crates/tiled).
+To easily write out the Rust code for the levels, we will use the [`quote`](https://crates.io/crates/quote) crate.
+In order to have access to the `TokenStream` type, we will also need [`proc-macro2`](https://crates.io/crates/proc-macro2).
+Include this in your `Cargo.toml`
+
+```toml
+[build-dependencies]
+quote = "1"
+proc-macro2 = "1"
+tiled = "0.14.0"
+```
+
+# `build.rs`
+
+The `build.rs` file placed in the manifest directory will be run before the compilation of your game.
```

Comment:
Maybe something along the lines of 'same directory as the Cargo.toml' rather than 'manifest directory'?